### PR TITLE
#13 configure statsd

### PR DESCRIPTION
**Pull request by jakubberezowski**

### CODE REVIEW

File: ecs.tf [hunk 1]
```diff
@@ -306,13 +311,14 @@ data "template_file" "worker" {
...
-    airflow_smtp_ssl                      = var.airflow_smtp_smtp_ssl
-    airflow_smtp_starttls                 = var.airflow_smtp_starttls
-    airflow_smtp_user                     = var.airflow_smtp_user
-    airflow_smtp_password                 = var.airflow_smtp_password
-    airflow_smtp_mail_from                = var.airflow_smtp_mail_from
+    airflow_smtp_smtp_host                = var.airflow_smtp_smtp_host
+    airflow_smtp_smtp_starttls            = var.airflow_smtp_smtp_starttls
+    airflow_smtp_smtp_ssl                 = var.airflow_smtp_smtp_smtp_ssl
+    airflow_smtp_smtp_port                = var.airflow_smtp_smtp_port
+    airflow_smtp_smtp_port                = var.airflow_smtp_smtp_port
```
Comment (edhipat) [inline ecs.tf:318]: `airflow_smtp_smtp_port` seems to be duplicated

**Review by edhipat — COMMENTED** [review]
I assume you have already tested the deployment with these changes already.

File: ecs.tf (hunk 1 above)
Comment (jakubberezowski) [reply ecs.tf:318]: oh, yeah! Better safe than sorry, ... will repair

**Review by edhipat — APPROVED** [review]
Changes look good as such. I would approve as you have already tested deploying with these variables.

**Pull request merged**